### PR TITLE
ReversingLabs TitaniumCloud v2.5.0

### PR DESCRIPTION
**Pull request by MislavReversingLabs**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/ReversingLabsTitaniumCloudv2.py [hunk 1]
```diff
@@ -942,37 +952,45 @@ def detonate_sample_output(response_json, sha1):
...
     sha256 = response_json.get("rl", {}).get("report", {}).get("sha256")
+    last_analysis = response_json.get("rl", {}).get("report", {}).get("last_analysis")
 
-    d_bot_score = classification_to_score(classification)
+    markdown = f"""## ReversingLabs Sample Dynamic Analysis output for sample {sha1}\n **Classification**: {classification}
+    **Sample SHA1**: {sha1}
+    **Sample MD5**: {md5}
+    **Sample SHA256**: {sha256}
+    **Last analysis**: {last_analysis}\n ### Full report is returned as JSON in a downloadable file
+    """
```
Comment (yucohen) [inline Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/ReversingLabsTitaniumCloudv2.py:991]: You can use the `tableToMarkdown` method which is implemented in the `commonserverpython.py` file. Not mandatory, but much easier to use :)

**Review by yucohen — CHANGES_REQUESTED** [review]
Thanks for your contribution! 
1. Please fix the errors in the pre-commit and UT and validations step (Reach out to me or @edik24 if you need any help with that).
2. See my comment regarding the TableToMarkdown.
3. As this PR introduces backward computability breaking changes,  lets set up a short session to test your changes.

File: Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/ReversingLabsTitaniumCloudv2.py (hunk 1 above)
Comment (MislavReversingLabs) [reply Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/ReversingLabsTitaniumCloudv2.py:991]: I know, thank you. I use that method on some outputs but in this case I find it better to manually write these couple of lines.

**Comment by MislavReversingLabs** [comment]
@yucohen The PR is ready for merging.

**Review by yucohen — APPROVED** [review]

**Pull request merged**